practice-two/src/components/ConfirmDeletePopup/ConfirmDeletePopup.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import './ConfirmDeletePopup.css';
+import { Button } from '@components';
+import { BUTTON_VARIANTS } from '@constants';
+
+const ConfirmDeletePopup = ({
+  onToggleDeletePopup,
+  onDeleteCustomer,
+}) => {
+  return (
+    <div className='confirm__popup'>
+      <div className='confirm__content'>
+        <p className='confirm__title'>Do you want to delete this customer?</p>
```

Comment:
this is confirm delete popup so it should be: Are you sure to delete customer?

Reply:
![image](https://github.com/datdragon777/React-Training/assets/74806826/4e98f4b4-d4bd-45b8-90bc-61ec72a46a93)